config.py

```diff
@@ -1527,6 +1528,8 @@ def get_t1_from_meeg(bids_path):
 VERSION = '0.1.dev0'
 CODE_URL = 'https://github.com/mne-tools/mne-bids-pipeline'
 
+os.environ['MNE_BIDS_STUDY_SCRIPT_PATH'] = str(__file__)
```

Comment:
Needs to be set for `config.py` because it's not passing through `failsafe_run()`, which does this export for all other scripts